doc/contributing/docs/style.rst

```diff
@@ -23,6 +23,44 @@ of the same databases."
 Correct usage: "Replication allows multiple Tarantool *instances* to work on
 copies of the same databases."
 
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Express one idea in a sentence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Say exactly one thing in a sentence.
+If you want to define or clarify something, do it in a separate sentence.
+Simple sentences are easier to read, understand and translate.
+
+..  rst-class:: table-example
+..  container:: table
+
+    ..  rst-class:: left-align-column-1
+    ..  rst-class:: left-align-column-2
+
+    ..  list-table::
+        :header-rows: 1
+
+        *   *   Don't
```

Comment:
it more readable for me when rows and columns marked differently, for example
```suggestion
        *   -   Don't
```

Reply:
That's a great idea, I like it.